libr/bin/pdb/pdb_downloader.c

```diff
@@ -106,7 +106,7 @@ void init_pdb_downloader(SPDBDownloaderOpt *opt, SPDBDownloader *pd) {
 	pd->download = download;
 }
 
-void deinit_pdb_downloader(SPDBDownloader *pd) {
+R_API void deinit_pdb_downloader(SPDBDownloader *pd) {
```

Comment:
symbols not prefixed with r_ should be public api and therefor dont use the R_API thing